spec/javascripts/health-care/components/AddressState.spec.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactTestUtils from 'react-addons-test-utils';
+import AddressState from
+    '../../../../_health-care/_js/components/personal-information/AddressState';
+
+describe('<AddressState>', () => {
+  let component = null;
+
+  beforeEach(() => {
+    component = ReactTestUtils.renderIntoDocument(
+      <AddressState state={{ state: 'CA' }} onUserInput={(_unused) => {}}/>
```

Comment:
Actually, I thought about this, and I think it makes more sense. Certainly makes our code cleaner to be using the same prop name. Let's do `value`! We just have to make sure we change it in the component itself and not just in the specs.